autopush-common/src/db/bigtable/bigtable_client/merge.rs

```diff
@@ -191,7 +191,10 @@ impl RowMerger {
 
         let cell = &mut row.cell_in_progress;
         if chunk.has_family_name() {
-            cell.family = chunk.take_family_name().get_value().to_owned();
+            chunk
+                .take_family_name()
+                .get_value()
+                .clone_into(&mut cell.family);
```

Comment:
Same as above? Reduces ownership transfers by allocating a mutable ref inside the `clone_into` call?